### PR TITLE
IPBlogPost first draft

### PR DESCRIPTION
**Pull request by latifahHX**
### Blog title
New Kids on the Block
### Blog description
An introduction of the new IPs
#### Author
John, Latifah, Mitchell, Rosie
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [ ] I have checked for grammatical errors
- [ ] I have checked for spelling errors
- [ ] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

File: _posts/2017-08-30-new-kids-on-the-block.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "New Kids on the Block"
+author: rosie_latifah_john_mitchell
+tags: IP, Tech, Profiles
+categories: meetTheTeam
+excerpt:comments: true Each year Holiday Extras take on several industrial placement (IP) students as web developers; this year they hired 4 to become part of the web team. Here’s a short summary of what each wants to achieve in their year at HX and how their first few weeks have been so far!
+--
+
+Each year Holiday Extras take on several industrial placement (IP) students as web developers; this year they hired 4 to become part of the web team. Here’s a short summary of what each wants to achieve in their year at HX and how their first few weeks have been so far!
```
Comment (mattsaunby) [inline _posts/2017-08-30-new-kids-on-the-block.md]: I'm being a bit picky but this feels a bit like it contradicts itself, you first say that each year HX takes on IP students and then say that this year they hired 4 to become part of the web team like its not the norm.
I would reword it to - "...students as web developers in their web team. Here's a short summary of what each of us wants to achieve...."

File: _posts/2017-08-30-new-kids-on-the-block.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Rosie Watson
+
+![rosiewatson.png]({{ site.baseurl }}/assets/img/rosiewatson.png)
+
+**Pod:** Euro
+
+**Goals for the year** 
+There are many reasons why I chose to do a year in industry, and specifically why I chose to apply to do it at Holiday Extras. I want use this year to increase my programming abilities and broaden the types of technologies I have been exposed too, as I had never had much experience outside of my course. Along with this I am keen to learn how tech companies are run and how many large systems can be maintained and improved at such a high rate. I also hope to improve upon the soft skills I currently have to gain more experience; such as giving presentations and writing formal documentation.
```
Comment (mattsaunby) [inline _posts/2017-08-30-new-kids-on-the-block.md]: Second paragraph - "I want use this year" needs to have the word "to" in there

File: _posts/2017-08-30-new-kids-on-the-block.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+
+**How the first few weeks went:**
+
+The first few weeks consisted of lots of workshops (to give us a certain level of knowledge to work off of) and rotating through 3 different pods (for me it was Euro, Mobile and Integration) to get a understanding of what each team is working on and what their long term goals are. This was a great experience and really helped me understand how all of the different pods worked together to maintain the sites and improve the experience for customers.
```
Comment (mattsaunby) [inline _posts/2017-08-30-new-kids-on-the-block.md]: More of a personal preference but I never think the phrase "off of" reads very well, maybe try "...to work from" instead?

File: _posts/2017-08-30-new-kids-on-the-block.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+**Pod:** Booking
+
+**Goals for the year:** 
+
+My goals for the year include progressing with my overall coding ability with a focus on being able to work across multiple languages and frameworks, allowing me to become more confident in the work I produce and broaden my skillset for the future. My second objective is to take a more active role in terms of speaking in front of people and taking the lead on projects, which HX seems to offer a lot of opportunities to work on.
+
+**How the first few weeks went:**
+
+Starting work at HX has been a new and exciting experience. Having worked for a few years before starting University I had come accustomed to working environments where you are expected to just show up, do your work, and then leave again. Coming to HX has shown me a whole new work lifestyle that fully surpasses what I thought work could be.
```
Comment (mattsaunby) [inline _posts/2017-08-30-new-kids-on-the-block.md]: I think it should be "become accustomed to" instead of "come"

File: _posts/2017-08-30-new-kids-on-the-block.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+I’ve been learning so much and it’s fantastic. The first few weeks sped by; our task was to understand the roles and objectives of different pods by rotating into one per week. I joined the Customer Experience, Payment, and Booking pods - incidentally, the latter is the one that I have finally settled into, and so far I am loving it! 
+
+There have been many systems to learn about, and many languages that I hadn’t heard about prior to coming here (who knew that Javascript could be so versatile?) It has been a lot to absorb, but I am thrilled by the challenge of learning how to use each of these proficiently and making a solid contribution towards fulfilling customer needs.
+Now, I am actively working on completing tasks in the sprints while maintaining Holiday Extras’ coding standards. There is a lot to take in, but I am confident that with the support from everyone here, I’ll know everything I need.  
+
+
+---
+
+Well, that's it from them! We will keep you posted so that you can follow them throughout the year!
```
Comment (mattsaunby) [inline _posts/2017-08-30-new-kids-on-the-block.md]: I deliberately used the word "us" in an example on the first paragraph, I think otherwise it looks like it's not posted by you guys, make it a bit more personal and refer to yourselves as "us" instead :)

**Comment by mattsaunby** [comment]
@latifahHX - I know what you mean about the first paragraph, maybe try re-writing it completely, maybe along the lines of - "Hey, we're the new kids on the block at Holiday Extras, or maybe better known as Industrial Placement Students (IPS) that the company take on every year to work in their web team for experience. Here's a short summary.......etc"

File: _posts/2017-08-30-new-kids-on-the-block.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "New Kids on the Block"
+author: rosie_latifah_john_mitchell
+tags: IP, Tech, Profiles
+categories: meetTheTeam
+excerpt: Each year Holiday Extras take on several industrial placement (IP) students as web developers in their web team. Here's a short summary of what each of us want to achieve in our year at HX and how our first few weeks have been so far!
+comments: true 
+--
```
Comment (DuncanFenning) [inline _posts/2017-08-30-new-kids-on-the-block.md]: Think this needs one more -, this will break your content

File: _posts/2017-08-30-new-kids-on-the-block.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+**How the first few weeks went:**
+
+Starting work at HX has been a new and exciting experience. Having worked for a few years before starting University I had become accustomed to working environments where you are expected to just show up, do your work, and then leave again. Coming to HX has shown me a whole new work lifestyle that fully surpasses what I thought work could be.
+
+During my first few weeks I have been getting acclimatised to what and how people do things here. New tech and languages have been the focus of my work and understanding what it is like to be a part of a team/pod. HX encourages you to ask questions without being judged which has helped me feel more comfortable in the fact that it is ok to be wrong. This has already helped with progressing my knowledge and skills further as it has reduced my worry of not knowing everything.
+
+## Latifah Parara
+
+![latifahparara.png]({{ site.baseurl }}/assets/img/latifahparara.png)
```
Comment (DuncanFenning) [inline _posts/2017-08-30-new-kids-on-the-block.md]: latifah's image is `.JPG` so this will need updating otherwise the image path will be broken

File: _posts/2017-08-30-new-kids-on-the-block.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "New Kids on the Block"
+author: rosie_latifah_john_mitchell
```
Comment (DuncanFenning) [inline _posts/2017-08-30-new-kids-on-the-block.md]: This author wont work unless you add a new member with this author's name into `_data/members.yml` with a corresponding gravatar id

**Comment by DuncanFenning** [comment]
Changes look good to me. Ran locally + everything working as expected from a dev point of view 👍

**Comment by mattsaunby** [comment]
Yep looks good now, have approved :+1:
